src/delphi/vcl.castlecontrol.reg.pas

```diff
@@ -0,0 +1,45 @@
+{
+  Copyright 2022-2023 Michalis Kamburelis.
+
+  This file is part of "Castle Game Engine".
+
+  "Castle Game Engine" is free software; see the file COPYING.txt,
+  included in this distribution, for details about the copyright.
+
+  "Castle Game Engine" is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+
+  ----------------------------------------------------------------------------
+}
+
+{ Control with OpenGL context on a Delphi VCL form. }
+unit vcl.castlecontrol.reg;
+
+{$I castleconf.inc}
+
+interface
+
+uses SysUtils, Classes, Vcl.Controls, Vcl.ExtCtrls, Types,  WinApi.Messages,
+  CastleGLVersion, CastleGLUtils,
+  CastleInternalContextBase, CastleInternalContextWgl, CastleInternalContainer,
+  CastleVectors, CastleKeysMouse, VCL.CastleControl;
+
+{ Designtime Only }
```

Comment:
No need for this comment `{ Designtime Only }`. It should already be said at unit header, as per previous review comment.

Same thing for `{ Designtime Only }` in implementation.